utilities/_layout.scss

```diff
@@ -0,0 +1,36 @@
+//Full responsive layout based on design system
+//Todo: amend spacing calc for between 414px and 599 as some misalignment
```

Comment:
I'm not seeing this, was it browser-specific?

Reply:
was a weird glitch, no longer present